src/mixed.js

```diff
@@ -477,15 +477,19 @@ const proto = (SchemaType.prototype = {
   },
 
   describe() {
-    let next = this.clone();
+    const next = this.clone();
 
     return {
       type: next._type,
       meta: next._meta,
       label: next._label,
       tests: next.tests
-        .map(fn => fn.TEST_NAME, {})
+        .map(fn => fn.TEST_NAME)
         .filter((n, idx, list) => list.indexOf(n) === idx),
+      testsParams: next.tests.reduce(
```

Comment:
i wonder if it's better to make tests a map like this with a value like `{ name, params }`?

Reply:
that makes sense and it was my first thought. The reason I go with `testsParams` is to avoid breaking change. If it's ok for you to make it as a breaking change than I can change, it's really small change

Reply:
ok, now `params` are with `tests`, can you review? Thanks